packages/core-components/src/components/product-list/SortMobile.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+import { CaretDownIcon } from '@/icons';
+
+import { FilterItemType } from './FilterItem';
+
+type SortMobileProps = {
+  sortItems: FilterItemType;
+  onClickApplySort?: (sortData: string[]) => void;
+};
+
+const SortMobile: React.FC<SortMobileProps> = ({
+  sortItems,
+  onClickApplySort,
+}) => {
+  const formatSortValue = (filterType: string, optionId: string) => {
+    return `${filterType}=${optionId}`;
```

Comment:
why do we need to use this format a. I'm guessing because you combine both filter logic and sort logic into one component. We are violating Single Responsible principle here, anh.

Reply:
this func must be global function, can called by other component and page. But as you design, I don't see where I can place it so it has been placed in this component and you in it's self

It prevent mistake when you want return value but you write '-' instead of '='